mgmtsystem_action/reports/mgmtsystem_action_report.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+from openerp import tools
+
+
+class MgmtsystemtActionReport(models.Model):
+    """Management System Action Report."""
```

Comment:
What is the advantage of creating a separate report, instead of a simple Pivot over the Actions model?